self-tests.yml

```diff
@@ -421,6 +421,13 @@ functions:
           set -o xtrace
           rm evergreen/bin/output.*
           rm evergreen/bin/jstests/*.xml
+  
+  write-repo-name-to-downstream_expansions.yaml-for-pine:
```

Comment:
this seems a bit long / verbose, could we shorten it to something like: `write-downstream-expansions-for-pine`